tests/upgradePMM_test.js

```diff
@@ -574,27 +570,6 @@ if (versionMinor >= 16) {
     },
   );
 
-  Scenario(
```

Comment:
Could we have a task to re-enable this? or this will check is not needed any more?

Reply:
please cherry pick this to RC branch also

Reply:
done, thanks